modules/virt-example-snippets-nmstate-IP-management.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-snippets-nmstate-IP-management_{context}"]
+= Example snippets: IP management
+
+The following are example configuration snippets for different methods of IP management. 
```

Comment:
Maybe reword this as "The following are configuration examples for different methods of IP management"?

Reply:
I agree this could be reworded, but I'd suggest something like "The following example configuration snippets demonstrate different methods of IP management."

Reply:
:alf: